viz/charts/linkerd-viz/templates/grafana.yaml

```diff
@@ -30,6 +30,11 @@ data:
     check_for_updates = false
     [panels]
     disable_sanitize_html = true
+    [log]
+    mode = console
+    [log.console]
+    format = {{ .Values.grafana.logFormat | default "text" }}
```

Comment:
Can you try adding a bit more logic here in order for this to be compatible with `.Values.defaultLogFormat` (e.g. converting "plain" to "text")?

Reply:
Done! (See prometheus bit, as I think the way I did might still be changed, depending on the desired system behavior and configuration possibilities)